core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -482,43 +484,37 @@ public void unmount(AlluxioURI path, UnmountOptions options)
   }
 
   /**
-   * Alluxio URIs in the Filesystem don't need a scheme or host. Warn the user and throw an
+   * Checks an {@link AlluxioURI} for scheme and authority information. Warn the user and throw an
    * exception if necessary.
    */
-  protected static void UriCheck(AlluxioURI uri) {
-    boolean logWarning = false;
+  private static void uriCheck(AlluxioURI uri) {
     if (uri.hasScheme()) {
-      if (uri.getScheme().compareTo("alluxio") == 0
-          || uri.getScheme().compareTo("alluxio-ft") == 0) {
-        logWarning = true;
+      if (uri.getScheme().equals(Constants.SCHEME) || uri.getScheme().equals(Constants.SCHEME_FT)) {
+        LOG.warn(String.format("The URI schemes \"%s\" or \"%s\" are not required in URIs passed to"
```

Comment:
The `LOG.x()` methods already support interpolation more efficiently (if the user sets log level to ERROR only, `LOG.warn("{}", x)` skips performing the interpolation).

```java
LOG.warn("The URI schemes \"{}\" or \"{}\" are not required in URIs passed to" 
    + " the Alluxio Filesystem client", Constants.SCHEME, Constants.SCHEME_FT);
```

Reply:
Wasn't aware of this, thanks!